src/corehost/cli/deps_resolver.cpp

```diff
@@ -303,25 +313,47 @@ bool deps_resolver_t::probe_deps_entry(const deps_entry_t& entry, const pal::str
     return false;
 }
 
-bool report_missing_assembly_in_manifest(const deps_entry_t& entry)
+bool report_missing_assembly_in_manifest(const deps_entry_t& entry, bool isNotError = false)
 {
-    trace::error(_X(
-        "Error:\n"
-        "  An assembly specified in the application dependencies manifest (%s) was not found:\n"
-        "    package: '%s', version: '%s'\n"
-        "    path: '%s'"),
-        entry.deps_file.c_str(), entry.library_name.c_str(), entry.library_version.c_str(), entry.relative_path.c_str());
+    bool showManifestListMessage = !entry.runtime_store_manifest_list.empty();
 
-    if (!entry.runtime_store_manifest_list.empty())
+    if (entry.asset_type == deps_entry_t::asset_types::resources)
```

Comment:
The next elseif logs as `warning`, not `info`. So there are three cases: 1) log as `info` for resources, 2) log as `warning` for "apphost" case also discussed below, 3) log as `error` for all other cases. False (which will cause app start to fail) is returned only for the error case.